readme.md

```diff
@@ -40,8 +40,8 @@ Why does the following code snippet throw an error, and what would fix the error
 var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
-
-> Your answer...
+the .style() prototype takes the style and modification as an argument.
+theBody.style(background-color = "red");
```

Comment:
No, but good guess! [Take a look at the solution.](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-3)